src/lib/y2storage/devicegraph.rb

```diff
@@ -90,6 +90,13 @@ class Devicegraph
     #   @return [Device]
     storage_forward :find_device, as: "Device"
 
+    # @!method remove_device(device)
+    #   Removes the device from the devicegraph
+    #
+    #   @param device [Device]
```

Comment:
Strictly speaking, I believe this would also work receiving a sid as argument, not only when receiving the Device object.